### DISCUSSION TITLE
加了print-memory-usage使用xmake编译不输出日志

### DISCUSSION THREAD

**Discussion by PeakRacing** [xmake-io/xmake#2916] [Q&A]
使用xmake编译：
![image](https://user-images.githubusercontent.com/24759100/195389162-106a9c88-affc-46a0-ae79-098af8edd173.png)
将此命令单独执行：
![image](https://user-images.githubusercontent.com/24759100/195389246-716b2478-e5ab-41bc-83ff-c8c00becc4cb.png)
希望使用xmake编译的情况下也输出此日志

**Comment by waruqi** [comment]
再试试

xmake update -s dev
xmake -v

**Reply by PeakRacing** [reply]
![image](https://user-images.githubusercontent.com/24759100/195497107-bbe90a79-f313-4c1a-b370-c242e452cd31.png)
有了

**Comment by waruqi** [comment]
可以试下这个 patch，可以配置 policy 强制启用，即使在非 verbose 模式
